tests/acceptance/click-add-different-project-map-types-test.js

```diff
@@ -0,0 +1,65 @@
+import { module, test } from 'qunit';
+import {
+  visit,
+  click,
+  fillIn,
+  currentURL,
+} from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+
+import mapboxGlLoaded from '../helpers/mapbox-gl-loaded';
+import mapboxGlDrawReady from '../helpers/mapbox-gl-draw-ready';
+import plotVerticies from '../helpers/plot-verticies';
+
+module('Acceptance | user can click to add different project map types', function(hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+
+  test('user can click to add different project map types', async (assert) => {
+    server.createList('project', 10);
+
+    await visit('/projects/new');
```

Comment:
Since this just tests adding a new map, my only feedback would be that you could just say:

`await visit('/projects/10');` 

This way we go directly to the project without duplicating the map creation test.